tldw_Server_API/app/api/v1/endpoints/outputs_templates.py

```
@@ -303,12 +306,21 @@ async def preview_output_template(
         if payload.item_ids:
             items = _build_items_context_from_media_ids(media_db, payload.item_ids, payload.limit)
         elif payload.run_id is not None:
-            mids = _select_media_ids_for_run(media_db, payload.run_id, payload.limit)
-            items = _build_items_context_from_media_ids(media_db, mids, payload.limit) if mids else []
+            items = []
+            try:
+                rows, _ = db.list_content_items(run_id=payload.run_id, page=1, size=payload.limit)
+                items = build_items_context_from_content_items(rows)
+            except Exception as exc:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)